test/test.js

```diff
@@ -8,149 +8,136 @@ var steam = require("steam"),
-                console.log("Node-dota2 unready.");
-            });
-            Dota2.on("chatMessage", function(channel, personaName, message) {
-                // util.log([channel, personaName, message].join(", "));
-            });
-            Dota2.on("guildInvite", function(guildId, guildName, inviter) {
-                // Dota2.setGuildAccountRole(guildId, 75028261, 3);
-            });
-            Dota2.on("unhandled", function(kMsg) {
-                util.log("UNHANDLED MESSAGE " + kMsg);
-            });
-            // setTimeout(function(){ Dota2.exit(); }, 5000);
-        }
-    },
-    onSteamServers = function onSteamServers(servers) {
-        util.log("Received servers.");
-        fs.writeFile('servers', JSON.stringify(servers));
-    },
-    onSteamLogOff = function onSteamLogOff(eresult) {
-        util.log("Logged off from Steam.");
-    },
-    onSteamError = function onSteamError(error) {
-        util.log("Connection closed by server.");
-    };
+            }
+
+            // ----------------------------------
+
+            // LOBBY
+
+            var creatingLobby = 0;
+            var leavingLobby = 0;
+
+            if(creatingLobby == 1){ // sets only password, nothing more
+                var lobbyPassword = "ap";
+                var lobbyName = "Lobby Name";
+
+                Dota2.createPracticeLobby(lobbyPassword, lobbyName, function(err, data){
+                    // util.log(JSON.stringify(data));
+                });
+            }
+
+            if(leavingLobby == 1){
+                setTimeout(function(){
+                    Dota2.leavePracticeLobby(function(err, data){
+                        // util.log(JSON.stringify(data));
+                    });
+                }, 10000);
+            }
+
+            // ----------------------------------
+        });
+
+        Dota2.on("unready", function onUnready() {
+            util.log("Node-dota2 unready.");
+        });
+
+        Dota2.on("chatMessage", function(channel, personaName, message) {
+            util.log("[" + channel + "] " + personaName + ": " + message);
+        });
+
+        Dota2.on("unhandled", function(kMsg) {
+            util.log("UNHANDLED MESSAGE #" + kMsg);
+        });
+    }
+},
+onSteamServers = function onSteamServers(servers) {
+    util.log("Received servers.");
+    fs.writeFile('servers', JSON.stringify(servers));
+},
+onSteamLogOff = function onSteamLogOff(eresult) {
+    util.log("Logged off from Steam.");
+},
+onSteamError = function onSteamError(error) {
+    util.log("Connection closed by server.");
+};
 
 steamUser.on('updateMachineAuth', function(sentry, callback) {
-        fs.writeFileSync('sentry', sentry.bytes)
-        util.log("sentryfile saved");
+    fs.writeFileSync('sentry', sentry.bytes)
+    util.log("sentryfile saved");
+
     callback({ sha_file: crypto.createHash('sha1').update(sentry.bytes).digest() });
 });
 
-
-// Login, only passing authCode if it exists
 var logOnDetails = {
```

Comment:
should use process.env.  Either use .env and dotenv to load or have user pass them in on the command line.